README.md

```diff
@@ -7,12 +7,34 @@
 
 ## How it works in our case
 
-We have [softwarefactory-project-zuul](https://github.com/apps/softwarefactory-project-zuul) GitHub App [installed to all packit-service repos](https://github.com/organizations/packit-service/settings/installations).
+We have [softwarefactory-project-zuul](https://github.com/apps/softwarefactory-project-zuul)
+GitHub App [installed to all packit-service repos](https://github.com/organizations/packit-service/settings/installations).
 We have a separate [packit-service tenant in Zuul](https://softwarefactory-project.io/zuul/t/packit-service/status).
-It uses the configuration from this repository, see especially [jobs.yaml](https://github.com/packit-service/packit-service-zuul/blob/main/zuul.d/jobs.yaml) and [pipelines.yaml](https://github.com/packit-service/packit-service-zuul/blob/main/zuul.d/pipelines.yaml) in [zuul.d/](https://github.com/packit-service/packit-service-zuul/tree/main/zuul.d).
-We can easily use those pipelines & jobs from other repositories so if you have a job which could be used by more repositories, you can add it to [jobs.yaml](https://github.com/packit-service/packit-service-zuul/blob/main/zuul.d/jobs.yaml).
-
-If you want to enable Zuul in another packit-service repository, just create a `.zuul.yaml` and define `noop` job inside to test the connection, see [example](https://github.com/packit-service/packit-service-zuul/pull/8/files).
+It uses the configuration from this repository, see especially
+[jobs.yaml](https://github.com/packit-service/packit-service-zuul/blob/main/zuul.d/jobs.yaml)
+and [pipelines.yaml](https://github.com/packit-service/packit-service-zuul/blob/main/zuul.d/pipelines.yaml)
+in [zuul.d/](https://github.com/packit-service/packit-service-zuul/tree/main/zuul.d).
+We can easily use those pipelines & jobs from other repositories so if you have
+a job which could be used by more repositories, you can add it to
+[jobs.yaml](https://github.com/packit-service/packit-service-zuul/blob/main/zuul.d/jobs.yaml).
+
+To enable Zuul in another repository, add it to
+[our tenant config](https://softwarefactory-project.io/cgit/config/tree/resources/tenant-packit-service.yaml)
+by following [The Config repository](https://softwarefactory-project.io/docs/user/config_repo.html)
+and [Contribute to a project](https://softwarefactory-project.io/docs/user/index.html).
+Specifically:
+
+- Add your ssh key to https://softwarefactory-project.io/r/settings/#SSHKeys if it's not already there.
```

Comment:
Did you manage to actually make this work? Following the link after logging in through Github throws Forbidden.

Reply:
hmm, I never got Forbidden in the webui, so you might be seeing a different issue after all

Reply:
It seems like there is an issue when creating new user in this gerrit. Perhaps adds the `softwarefactory-operations-team@redhat.com` point of contact for zuul and gerrit issues.

Reply:
added, thanks Tristan